pisek/task_jobs/data/manager.py

```diff
@@ -106,6 +98,40 @@ def _get_jobs(self) -> list[Job]:
 
         return jobs
 
+    def _report_unused_inputs(self, unused_inputs: Iterable[InputInfo]) -> None:
+        inputs = list(sorted(unused_inputs, key=lambda inp: inp.name))
+        if self._env.config.checks.no_unused_inputs and len(inputs):
+            if self._env.verbosity <= 0:
+                self._warn(
+                    f"{len(inputs)} unused input{'s' if len(inputs) >= 2 else ''}. "
+                    f"({self._short_inputs_list(inputs)})"
+                )
+            else:
+                for inp in inputs:
+                    self._warn(
+                        f"Unused {'generated' if inp.is_generated else 'static'} input: '{inp.name}.in'"
+                    )
+
+    def _report_not_included_inputs(
+        self, not_included_inputs: Iterable[InputInfo]
+    ) -> None:
+        inputs = list(sorted(not_included_inputs, key=lambda inp: inp.name))
+        if self._env.config.checks.all_inputs_in_last_subtask and len(inputs) > 0:
```

Comment:
Empty collections are falsy:
```suggestion
        if self._env.config.checks.all_inputs_in_last_subtask and inputs:
```